pages/index.js

```diff
@@ -0,0 +1,114 @@
+                            </p>
+                        </div>
+                        <div className={css.homesection2card}>
+                            <CenterFocusWeakIcon className={css.blackSectionIcons}/>
+                            <p className={css.centerText}>Focus on Creating</p>
+                            <p className={css.centerText}>The best way to get better at programming is to write code yourself, and learn from well-written code by others.                                
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            {/* Team Section */}
+            <div className={homecss.team_section}>
+                {/* <Member /> */}
+                <div className={css.teambackgroundimage}>
+                    <div className={css.topspace}>
+                        <h2 className={css.sectiontitle}>Teams</h2>
+                        <h4 className={css.sectionbodywhite}>Our valuable contributors from all over the world
+                        </h4>
+                        <div className={teamcss.ImageLists}>
+                            {teams.data.map(({id,login,html_url,avatar_url}) => (
+                                <Member key={id} login={login} html_url={html_url} avatar_url={avatar_url} />
+                            ))}
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className={homecss.service_section}>
+                    <div className={css.topspace}>
+                        <h2 className={css.sectiontitle}>Projects</h2>
+                        <h4 className={css.sectionbodygray}>Our world class projects which makes your life easier </h4>
+                        <div className={css.cardHolder}>
+                            <Card id="15924904" name="django-htk" description="A set of apps, utilities, middlewares, etc for Django" forks_count="" watchers_count="" language="" html_url="https://github.com/hacktoolkit/django-htk" />
+                            <Card id="15924904" name="xterm.js" description="A set of apps, utilities, middlewares, etc for Django" forks_count="" watchers_count="" language="" html_url="https://github.com/hacktoolkit/django-htk" />
+                            <Card id="15924904" name="django-htk" description="A set of apps, utilities, middlewares, etc for Django" forks_count="" watchers_count="" language="" html_url="https://github.com/hacktoolkit/django-htk" />
+                        </div>
+                        <Link href="/projects">
+                            <h2 className={css.seeMore}>See more projects...</h2>
+                        </Link>
+                    </div>
+            </div>
+        </Page>
+    );
+}
+
+export async function getStaticProps() {
+    const octokit = new Octokit({
+        auth: process.env.API_KEY,
```

Comment:
A couple of changes:

1. Make the env var `GITHUB_API_KEY` instead of `API_KEY`
2. This needs to be statically loaded from a JSON file, otherwise we won't be able to do `next export` to make this a static website and host it on GitHub static pages. Pretty much almost anything that uses `await` is not allowed for `next export` (except reading in of local system files, I believe). You can test it out and see if `next export` is allowed or not.
3. We should have a separate scripts directory that we can run that has a bunch of NodeJS scripts that can be run in the CLI, to save the JSON to static files (to a directory named something like `/static/data/team_members.json`)

Reply:
1. done

Reply:
2. first let's finalize the remaining pages and we will try to figure out solution for await... issue.. if that's Ok.

Reply:
3. that is a great solution. we might also need more API requests on the community page. so will do.